[2024-05-25T03:36:05.110+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T03:36:05.138+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v03.transform_data_response scheduled__2024-05-24T00:00:00+00:00 [queued]>
[2024-05-25T03:36:05.149+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v03.transform_data_response scheduled__2024-05-24T00:00:00+00:00 [queued]>
[2024-05-25T03:36:05.152+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-25T03:36:05.170+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data_response> on 2024-05-24 00:00:00+00:00
[2024-05-25T03:36:05.178+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_weather_workflow_DAG_v03', 'transform_data_response', 'scheduled__2024-05-24T00:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp8etebn2d']
[2024-05-25T03:36:05.182+0000] {standard_task_runner.py:91} INFO - Job 76: Subtask transform_data_response
[2024-05-25T03:36:05.185+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=169) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-25T03:36:05.186+0000] {standard_task_runner.py:63} INFO - Started process 177 to run task
[2024-05-25T03:36:05.257+0000] {task_command.py:426} INFO - Running <TaskInstance: my_weather_workflow_DAG_v03.transform_data_response scheduled__2024-05-24T00:00:00+00:00 [running]> on host 6ab40057742d
[2024-05-25T03:36:05.643+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khang' AIRFLOW_CTX_DAG_ID='my_weather_workflow_DAG_v03' AIRFLOW_CTX_TASK_ID='transform_data_response' AIRFLOW_CTX_EXECUTION_DATE='2024-05-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-24T00:00:00+00:00'
[2024-05-25T03:36:05.645+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T03:36:05.672+0000] {logging_mixin.py:188} INFO - /opt/***
[2024-05-25T03:36:05.674+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-25T03:36:05.676+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T03:36:05.700+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_weather_workflow_DAG_v03, task_id=transform_data_response, run_id=scheduled__2024-05-24T00:00:00+00:00, execution_date=20240524T000000, start_date=20240525T033605, end_date=20240525T033605
[2024-05-25T03:36:05.764+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T03:36:05.800+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-25T03:36:05.804+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
